doc/source/roadmap.rst

```diff
@@ -98,11 +98,13 @@ however; we need to make it faster and make it easier to compare performance of
 optimizers via plotting performance profiles.
 
 
-Linear programming enhancements
--------------------------------
-
-Recently all known issues with ``optimize.linprog`` have been solved.  Now we
-have many ideas for additional functionality (e.g. integer constraints, sparse
-matrix support, performance improvements), see gh-9269.
+Statistics enhancements
+-----------------------
 
+The `scipy.stats` enhancements listed in the :ref:`scipy-roadmap-detailed` are of
+particularly high importance to the project.
 
+- Improve the options for fitting a probability distribution to data.
+- Expand the set of hypothesis tests.  In particular, include all the basic
+  variations of analysis of variance.
+- Add confidence intervals for all statistical tests.
```

Comment:
There are several types of intervals besides confidence intervals: prediction intervals, tolerance intervals, HPD (Bayesian) intervals to name some, not sure if you want to add a list or expand the scope slightly.

Reply:
For now, I'm going to stick to confidence intervals for this pull request.  But this also falls in the category of a possible future addition.